content/ui/sessionTimeout.html

```diff
@@ -6,6 +6,7 @@
     <title>Document</title>
 </head>
 <body>
-    <h1> Your session has expired. Kindly logout from kids mode and start a new session if you want</h1>
+    <div style="display: flex; justify-content: center; align-items: center;"><h1> Your session has expired, kindly logout of kids mode or start a new session.</h1></div>
```

Comment:
change this text to
`Thats enough of the internet for today! Logout of kids mode to keep browsing`